packages/web-components/src/components/card/card.scss

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright IBM Corp. 2020
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '@carbon/ibmdotcom-styles/scss/components/card/index';
+@import '@carbon/ibmdotcom-styles/scss/components/feature-card/feature-card';
+@import '@carbon/ibmdotcom-styles/scss/components/link-with-icon/link-with-icon';
+@import '../../globals/scss/vars';
+
+:host(#{$dds-prefix}-card) {
```

Comment:
Just curious, should we be including :host / ::slotted in the styles package, and we keep the scss styling in the web-components package to a minimum?

Reply:
@asudoh @RaphaelAmadeu this is more of a question, if this is something we should think about or it isn't possible? I still am not that far into the training just wanted to get an understanding.

Reply:
@jeffchew It's in my personal roadmap. The blocker is how our styles (in `ibmdotcom-styles` package) are structured, that disallows us to reuse most of the style rules. One idea I have is limiting Sass nesting to pseudo elements only, and make stuffs work with less parent-child (ancestor/descendant) selectors.

Reply:
@asudoh sounds like a bunch of rework, but certainly we should think about it in order for us to find more reusability between the react and web-components package...